demos/paired_mrus_prostate/demo_data.py

```diff
@@ -67,8 +67,8 @@
     shutil.rmtree(MODEL_PATH)
 os.mkdir(MODEL_PATH)
 
-ZIP_PATH = "paired_mrus_prostate-ckpt"
-ORIGIN = "https://github.com/DeepRegNet/deepreg-model-zoo/raw/master/paired_mrus_prostate-ckpt.zip"
+ZIP_PATH = "paired_mrus_prostate_1"
```

Comment:
maybe a better name XD?

Reply:
:D this is really for my internal record...